ecr_api.py

```diff
@@ -197,35 +248,151 @@ def getApp(self, app_id):
 
         return returnObj
     
-    def listApps(self):
-        stmt = f'SELECT  BIN_TO_UUID(id), name, version FROM Apps'
+    def getAppField(self, app_id, field):
+        stmt = f'SELECT  BIN_TO_UUID(id), {field} FROM Apps WHERE BIN_TO_UUID(id) = %s'
+        print(f'stmt: {stmt} app_id={app_id}', file=sys.stderr)
+        self.cur.execute(stmt, (app_id, ))
+
+        returnFields = ["id", field]
+        returnObj={}
+        row = self.cur.fetchone()
+        i = 0
+        if row == None:
+            raise ErrorResponse(f'App {app_id} not found', status_code=HTTPStatus.INTERNAL_SERVER_ERROR)
+        for value in row:
+            print(f'value: {value}', file=sys.stderr)
+            returnObj[returnFields[i]] = value
+            i+=1
+
+        #decode embedded json
+        if field in ["inputs", "metadata"]:
+            if field in returnObj:
+                returnObj[field] = json.loads(returnObj[field])
```

Comment:
These lines are showing up as untested.  did you do that on-purpose?  If it is hard to test then I'm cool skipping it.